src/components/MultiHopTrade/components/MultiHopTradeConfirm/MultiHopTradeConfirm.tsx

```diff
@@ -142,9 +143,16 @@ export const MultiHopTradeConfirm = memo(() => {
             titleTranslation={
               isArbitrumBridgeWithdraw ? 'bridge.arbitrum.success.tradeSuccess' : undefined
             }
-            descriptionTranslation={
-              isArbitrumBridgeWithdraw ? 'bridge.arbitrum.success.withdrawComplete' : undefined
-            }
+            sellAsset={activeQuote?.steps[0].sellAsset}
```

Comment:
suggestion: l.140 checks should be `isTradeComplete && activeQuote`: if we don't have an `activeQuote` here, we have bigger problems

Reply:
Done in [b1a7cbb](https://github.com/shapeshift/web/pull/8240/commits/b1a7cbbefdcd14c05973b2565dd569c641205b9d)